### DISCUSSION TITLE
FR: Option to disable scroll wheel level changing, ability to Ctrl+Z to undo change.

### DISCUSSION THREAD

**Discussion by drgruney** [File-New-Project/EarTrumpet#637] [Ideas]
I have a lot of devices and programs going at the same time and moving up and down the list is a tad burdensome with the scroll bar being teensy. It's also natural reflect to see a list and hit the scroll wheel. Invariably I accidentally change an audio level and don't know what the previous setting was.

I would love being able to toggle scroll to set levels off so it just scrolls through my device/application list.

Additionally being able to "Undo" the last level change could be nice in fine tuning for levels out.

**Comment by vertigo220** [comment]
I didn't even realize there was a scroll bar... If you scroll the list with the mouse over the program icons or the numeric volume level (e.g. 100), it will actually scroll. It only changes the volume when over the slider.

That said, I would like an option to change this so it _only scrolls when hovering over the numeric volume level_. I'm frequently changing the volume because, as you said, it's just habit to scroll in order to, well, scroll. I think it would be a lot better if everything left of the volume number reacted to the scroll wheel by scrolling the list, and in order to change a specific volume you'd hover over the number and scroll, or just drag the bar. I realize, of course, not everyone would like this, which is why it would need to be an option. But it's definitely frustrating the way it's currently set up.

**Comment by wiretail** [comment]
Another vote from me.  An option of what the scroll wheel does would be excellent, one being to just scroll the list only.

I've never once tried or wanted to adjust the volume with the scroll wheel.  But I am constantly doing it by accident in my long list of audio devices.  And then either getting the mouse just right over the numbers or trying to get the scroll bar to appear to scroll the list.

**Comment by Deagan** [comment]
Yes, please! I use Volume2 to manage my volume with the scroll wheel, so the EarTrumpet feature conflicts with it. I would much rather disable it.

**Reply by vertigo220** [reply]
I'm probably misunderstanding your problem, since this issue only exists when ET's popup is open, but if that's all you're using that for, you may be able to do the same (and more) with 7+ Taskbar Tweaker (adjust volume by scrolling over taskbar and/or tray) and maybe won't have the same problem. And SoundSwitch offers some of the same functionality it appears Volume2 does, in case that would be an issue. No idea if those will help, but figured I'd mention them just in case.

**Reply by Deagan** [reply]
I just needed to be able to disable the EarTrumpet scroll-wheel volume changing.
This occurs while hovering over the system tray icon!

**Reply by vertigo220** [reply]
Ah, I didn't realize it did that. I keep it in the overflow area, so it's normally hidden, which also makes it so that feature only works when hovering over the ET icon, not the whole tray area. Granted it's an additional click to get to it, but might be worth doing that as a workaround to this issue.

**Comment by knapman** [comment]
Agree, this feature just leads to accidental adjustments, I never want the wheel to adjust sliders, I just want it to scroll the list, thanks to things like Steam and Oculus the device list is quite long, would love an option to disable this behaviour
